website/source/docs/builders/azure.html.md

```diff
@@ -221,6 +221,10 @@ Providing `temp_resource_group_name` or `location` in combination with `build_re
 
     CLI example `azure vm sizes -l westus`
 
+-   `async_resourcegroup_delete` (boolean) If you want packer to delete the temporary resource group asynchronously. Its a boolean value
+     and defaults to false. **Important** Setting this true would mean that your builds are faster however this is a very
+     small chance that the temporary resource group is not deleted by Azure.
```

Comment:
I would probably just say the builder does not wait for delete to complete.  If delete fails today the builder does not do anything other than report it.

Reply:
yup .. updated